### PR TITLE
portfolio-form-branch

### PR DESCRIPTION
**Pull request by abiodunraheem**
Create forms with HTML5 validations.
Understand the importance of UX.
Understand how to parse a Figma design to create a UI.
## Changes effected
action link changed.
maxlength of 30 was added to the name input field.
required attribute was added to the input.
maxlength of 500 was added to the textarea.
form button was moved inside the form tag.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
-            <div class="input-details">
-                <input type="email" placeholder="Email address">
-                <hr>
-            </div>
-            <div class="text-area">
-            <textarea name="Enter-text" id="enter-text" cols="41" rows="5" placeholder="Enter text here"></textarea>
-            </div>
-            <div class="get-in-touch"><a href="#">Get in touch</a></div>
-
+            <form action="#" method="post">
```
Comment (gemmen29) [inline index.html:208]: - Please change the action link to be similar to `https://formspree.io/f/{form_id}`.  Kindly use the site provided in the requirement of the project

File: index.html [hunk 2]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
+                <ul>
+                    <li class="input-details">
+                        <input type="text" placeholder="Full name">
```
Comment (gemmen29) [inline index.html:211]: Kindly add `maxlength="30"` for this input as it is in the project requirement

File: index.html [hunk 3]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
+                        <hr>
+                    </li>
+                    <li class="input-details">
+                        <input type="email" placeholder="Email address">
+                        <hr>
+                    </li>
+                    <li class="text-area">
+                    <textarea name="Enter-text" id="enter-text" cols="41" rows="5" placeholder="Enter text here"></textarea>
```
Comment (gemmen29) [inline index.html:219]: Please add the `required` attribute for all input as requested in the project requirement.

File: index.html [hunk 4]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
-            <div class="get-in-touch"><a href="#">Get in touch</a></div>
-
+            <form action="#" method="post">
+                <ul>
+                    <li class="input-details">
+                        <input type="text" placeholder="Full name">
+                        <hr>
+                    </li>
+                    <li class="input-details">
+                        <input type="email" placeholder="Email address">
```
Comment (gemmen29) [inline index.html:215]: Kindly add `maxlength="500"` for this input as it is in the project requirement

File: index.html [hunk 5]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
+                        <hr>
+                    </li>
+                    <li class="text-area">
+                    <textarea name="Enter-text" id="enter-text" cols="41" rows="5" placeholder="Enter text here"></textarea>
+                    </li>
+                </ul>
+            </form>
+            <button name="button" type="submit" class="btn-1">Get in touch</button>
```
Comment (gemmen29) [inline index.html:223]: Please move this button to be inside the `form` tag as it now not related to the form

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @abiodunraheem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -205,20 +205,22 @@ <h2>About me</h2>
...
+                    <li class="input-details">
+                        <input type="text" placeholder="Full name" maxlength="30" required>
+                        <hr>
+                    </li>
+                    <li class="input-details">
+                        <input type="email" placeholder="Email address" required>
+                        <hr>
+                    </li>
+                    <li class="text-area">
+                    <textarea name="Enter-text" id="enter-text" cols="41" rows="5" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (micheaol) [inline index.html:219]: [OPTIONAL]: Please add the `required` attribute to the `textarea`.

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @abiodunraheem , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**